### PR TITLE
Implement dependabot configuration

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

To automate the search for 3rd party component updates, the dependabot configuration has been added

## Type of change

- [x] CI system update

# How Has This Been Tested?

Every month will automatically create PRs for component updates

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+# Enable version updates for maven
+  - package-ecosystem: "maven"
+    directory: "/"
+    # Check for updates once a month
+    schedule:
+      interval: "monthly"
```
Comment (MoonkiHong) [inline .github/dependabot.yml:8]: Does it reasonable?

**Review by MoonkiHong — COMMENTED** [review]
PTAL. 😄 I need your opinion.

File: .github/dependabot.yml (hunk 1 above)
Comment (tdrozdovsky) [reply .github/dependabot.yml:8]: At the initial stage, I suggest monthly, as there may be a large number of automatically generated PRs.

**Review by MoonkiHong — APPROVED** [review]
+1

**Review by o-kopysov — APPROVED** [review]
LGTM

**Pull request merged**